src/Utils.php

```diff
@@ -579,7 +579,7 @@ function wf_crm_contact_label($n, $data = [], $html = 'escape') {
       $label = Html::escape($label);
     }
     if ($html == 'wrap') {
-      $label = Markup::create('<span class="contact-label number-' . $n . '">' . $label . '</span>');
+      $label = Markup::create($label);
```

Comment:
Looks like this label should be updated when the label on the Contact tab is changed as per https://github.com/colemanw/webform_civicrm/pull/506/files#diff-29604b8d3d680574d348a450de44692476847958c077b350b4dd553c4864f07dL582

The class was not loading before this PR too so merging this and we can handle this issue in a separate PR.